cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -3349,6 +3349,16 @@ readVerificationKey asType =
       first (Text.unpack . renderInputDecodeError) $
         deserialiseInput (AsVerificationKey asType) keyFormats (BSC.pack str)
 
+-- | Read a Bech32 or hex-encoded stake key.
+readStakeKeyHash :: Opt.ReadM (Hash StakeKey)
+readStakeKeyHash = Opt.eitherReader deserialiseFromHex
```

Comment:
This is called `readStakeKeyHash` because of the type and also because `pStakeAddress` already exists.  Should that be used instead?

```

pStakeAddress :: Parser StakeAddress
pStakeAddress =
    Opt.option (readerFromParsecParser parseStakeAddress)
      (  Opt.long "stake-address"
      <> Opt.metavar "ADDRESS"
      <> Opt.help "Target stake address (bech32 format)."
      )


parseStakeAddress :: Parsec.Parser StakeAddress
parseStakeAddress = do
    str <- lexPlausibleAddressString
    case deserialiseAddress AsStakeAddress str of
      Nothing   -> fail $ "invalid address: " <> Text.unpack str
      Just addr -> pure addr
```

Reply:
I think the name `readStakeKeyHash` follows the naming convention we have for other `ReadM` values.